docs/evaluate/how-it-works.mdx

```diff
@@ -0,0 +1,90 @@
+
+![Describing a Temporal Application - How It Works](/img/how-it-works/hiw-yourapplicationclient.png)
+
+### Temporal Application
+
+A Temporal application is the code you write, comprised of Workflow and Activities. 
+A Temporal Workflow defines your overall business logic. That business logic might involve moving money between bank accounts, processing orders, deploying cloud infrastructure, training an AI model, or something else entirely.
+The Worker runs the Workflow and Activity code.
+
+![Workflow Function Execution - How It Works](/img/how-it-works/hiw-workflowexecution.png)
+
+### Activity
+Activities represent code (either a functions or method) that handle tasks prone to errors or failure, such as network requests, calling a service, sending an email, or generating random numbers.
+Activities are invoked in Workflow code and the Temporal Service coordinates with the application to execute them.
+
+![Continuing a Workflow - How It Works](/img/how-it-works/hiw-continuingtheworkflow.png)
+
+### Temporal Service
+
+Temporal consists of two components, the [Temporal Service](/evaluate/major-components#temporal-service) and an SDK in your preferred language such as Go, .NET, Java, Python, PHP, or Typescript.
+The Temporal Service logs the progress and state of your application.
+Once the application finishes executing the Activity, it communicates with the Temporal Service, which records the Activity's completion in the Workflow's Event History.
+The Activity invocation is handled by the Temporal Service.
+The Temporal Service updates the Event History and creates a Task for executing the Activity.
+
+![Activity Execution - How It Works](/img/how-it-works/hiw-activityexecution.png)
+
+### Recovering from Failure
+
+A Temporal Workflow is basically a process or sequence of events you want to make sure will execute reliably.
+They can run—and keep running—for years, even if the underlying infrastructure fails.
+If the execution of your Workflow code crashes, Temporal automatically recreates its pre-crash state, allowing the Workflow to continue execution as if it never even happened.
+Temporal provides built-in solutions for timers, event sourcing, state checkpointing, retries, and timeouts, removing the necessity for custom implementations.
+
+![Recovering from Failure - How It Works](/img/how-it-works/hiw-recoveringfromfailure.png)
+
+### Concluding the Workflow
+
+In this example, there are two Activities. The second Activity is complete, but the Workflow function still has additional code to run.
+The Temporal Service creates a new Task requesting that the application run the additional code.
+
+![Concluding the Workflow - How It Works](/img/how-it-works/hiw-concludingtheworkflow.png)
+
+## Common Misconceptions
+
+The Temporal Service does not execute your code!
+
+Your application code, including Workers, Workflows, and Activities, lives on your computers. 
```

Comment:
Not always locally. How about: "Your application code, including Workers, Workflows, and Activities, runs on your own infrastructure, whether that's on local machines, servers, or cloud environments."